flyteadmin/pkg/manager/impl/executions/queues.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"math/rand"
 
+	"github.com/flyteorg/flyte/flyteadmin/pkg/errors"
```

Comment:
can we update to have all flyteorg repos without spaces between and alphabetized for linter?

Reply:
Not sure I understand this comment. The imports are managed by goimports that CI enforces.

Reply:
I will see why it is not automatically formatted.

Reply:
Strange `goimports` doesn't handle this and the other places.

Reply:
thanks!